atom/browser/native_window_views_win.cc

```diff
@@ -194,6 +194,8 @@ bool NativeWindowViews::PreHandleMSG(UINT message,
       if (prevent_default) {
         ::GetWindowRect(GetAcceleratedWidget(),
                         reinterpret_cast<RECT*>(l_param));
+        return true;  // Tells Windows that the Sizing is handled. If not true,
+                      // call to event.preventDefault() has no effect
```

Comment:
Is this comment still correct then?

Reply:
@nornagon you're right, it is partially correct, I'll change it immediately to "Tells Windows that the Sizing is handled.", without the second part.
Thanks for the review